content/influxdb/cloud/organizations/buckets/bucket-schema/_index.md

```diff
@@ -12,161 +12,222 @@ related:
+The examples below reference [**InfluxDB data elements**](/influxdb/cloud/reference/key-concepts/data-elements/). We recommend reviewing data elements, [**InfluxDB key concepts**](/influxdb/cloud/reference/key-concepts/), and [**elements of line protocol**](/influxdb/cloud/reference/syntax/line-protocol/#elements-of-line-protocol) if you aren't familiar with these concepts.
+
 {{% /note %}}
 
-- [Create a bucket schema](#create-a-bucket-schema)
+- [Create an explicit bucket and schema](#create-an-explicit-bucket-and-schema)
+  - [Create a bucket schema using the influx CLI](#create-a-bucket-schema-using-the-influx-cli)
+  - [Create a bucket schema using the InfluxDB HTTP API](#create-a-bucket-schema-using-the-influxdb-http-api)
+  - [Write valid schemas](#write-valid-schemas)
+- [View bucket schema type and schemas](#view-bucket-schema-type-and-schemas)
+  - [View schema type and schemas in the InfluxDB UI](#view-schema-type-and-schemas-in-the-influxdb-ui)
+  - [View schema type and schemas using the influx CLI](#view-schema-type-and-schemas-using-the-influx-cli)
+  - [View schema type and schemas using the InfluxDB HTTP API](#view-schema-type-and-schemas-using-the-influxdb-http-api)
 - [Update a bucket schema](#update-a-bucket-schema)
+	- [Update a bucket schema using the influx CLI](#update-a-bucket-schema-using-the-influx-cli)
+	- [Update a bucket schema using the InfluxDB HTTP API](#update-a-bucket-schema-using-the-influxdb-http-api)
+- [Troubleshoot bucket schema errors](#troubleshoot-bucket-schema-errors)
 - [Troubleshoot write errors](#troubleshoot-write-errors)
 
-## Create a bucket schema
-Use the `influx` CLI to set the schema-type and measurement schemas for your bucket:
-1. Create a bucket with the `schema-type` flag set to `explicit`.
+## Create an explicit bucket and schema
+
+To create an explicit bucket and schemas for your data, do the following:
+
+  1. If you haven't already, [create a bucket that enforces explicit schemas](/influxdb/cloud/organizations/buckets/create-bucket/#create-a-bucket-that-enforces-explicit-schemas).
+  2. [Create a bucket schema](#create-a-bucket-schema).
+
+### Create a bucket schema
+
+With an `explicit` bucket, you predefine *measurement schemas* with column names, tags, fields, and data types for measurements.
+A measurement schema has the following properties:
+
+- `name`: the measurement name. The name must match the [measurement column](/influxdb/cloud/reference/key-concepts/data-elements/#measurement) in your data, obey [naming rules](#write-valid-schemas), and be unique within the bucket.
+- `columns`: a list of *column definitions* for the measurement.
```

Comment:
```suggestion
- `columns`: List of *column definitions* for the measurement.
```

Reply:
But, [this](https://developers.google.com/style/articles):

> For ease of comprehension and translation, include a, an, and the in your writing.